bower.json

```diff
@@ -16,9 +16,10 @@
 		"jshint": "2.4.4",
 		"qunit": "1.18.0",
 		"qunit-assert-classes": "1.0.2",
-		"qunit-assert-close": "JamesMGreene/qunit-assert-close#v1.1.1",
+		"qunit-assert-close": "apsdehal/qunit-assert-close#amd-patch",
```

Comment:
Just released a `v2.0.0` version that should meet your needs.  Thanks for the ping, @scottgonzalez.